styles/globals.scss

```diff
@@ -0,0 +1,68 @@
+@use "/styles/variables" as variables;
+@use "/styles/variables" as mixins;
+
+@import url('https://fonts.googleapis.com/css2?family=Lora'); // Font Lora
+
+// Note: Font weight - https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight
+// Note: Ask about bold and bolder
+
+h1 { // Lora 32px bold
```

Comment:
Change these styling into mixins or class. This will reduce the amount of repeated code.